lib/response.js

```diff
@@ -822,6 +822,14 @@ res.get = function(field){
  */
 
 res.clearCookie = function clearCookie(name, options) {
+  if (options) {
+    if (options.maxAge) {
+      deprecate('res.clearCookie: Passing "options.maxAge" is deprecated and should be removed. Starting with the next major release of Express, this option will be ignored, as res.clearCookie will automatically set cookies to expire immediately. Please update your code to omit this option.');
```

Comment:
Should we call out `v5.0.0` here so that we can be sure to not accidentally miss confusing folks in the future who may not know this was landed in v4?

Reply:
Updating to that. I wasn't sure if we wanted to say v5 or not, but this makes sense.